### PR TITLE
Name custom attributes ICs nicely

### PR DESCRIPTION
**Pull request by Ladas**
Name custom attributes ICs nicely, so it shows up nicely in log and the graph.

### CODE REVIEW

File: app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb [hunk 1]
```diff
@@ -338,6 +338,7 @@ def initialize_custom_attributes_collections(manager, association, sections)
       @collections[[:custom_attributes_for, relation.model.base_class.name, section.to_s]] =
         ::ManagerRefresh::InventoryCollection.new(
           shared_options.merge(
+            :name                         => "#{association}_#{section}".to_sym,
```
Comment (cben) [inline app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb]: can you also add name in initialize_container_conditions_collection?

Comment (Ladas) [reply app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb]: yes, good idea

**Comment by Ladas** [comment]
@moolitayer @cben both custom attributes and conditions have name now

**Comment by moolitayer** [comment]
cc @cben

**Pull request merged**